.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: C/C++ CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-16.04, macos-latest, windows-latest]
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        # Using 2.8 results in cmake finding the 2.7 on the windows builder
+        # at C:/ProgramData/chocolatey/bin/python2.7.exe and this version seems
+        # to have some issues.
+        python-version: '2.7'
```

Comment:
no py3 available?

Reply:
wabt tests are currently broken under python3: https://github.com/WebAssembly/wabt/issues/1248

We need to fix that but thats a separate issue.